streamlit/mmm-explainer/README.md

```diff
@@ -0,0 +1,26 @@
+# MMM Visualization App with Streamlit
```

Comment:
Could we add in the README an explanation about how to deploy new functions once they are added to the `transformer.py` file? Imagine, tomorrow we add a new type of saturation, what are the steps to make it available here?

Reply:
This might be a bit tricky depending on what breaking changes occur in pymc-marketing API. However, for the current version in the requirements.txt - I believe I've addressed this ☺️